ConsistencyManager/ConsistencyManager.swift

```diff
@@ -57,6 +57,11 @@ public class ConsistencyManager {
      Periodically, the consistency manager cleans up after itself.
      This is a relatively quick process, and since the memory footprint of the library in general is small, it's not an important thing to worry about.
      Default is 5 mins. It will also clean up automatically on any memory warnings.
+
+     If you set this to zero, then garbage collection is considered disabled.
+     The ConsistencyManager will still attempt to clean up memory if the application receives a memory warning.
+
+     This variable is not thread-safe. You should only access this on the main thread.
      */
     public var garbageCollectionInterval: NSTimeInterval = 300
```

Comment:
Why not used a synchronized function to update this value and make it private? You can avoid the thread jump that way?